ansibullbot/triagers/ansible.py

```diff
@@ -2179,8 +2179,9 @@ def waiting_on(self, issuewrapper, meta):
         return {'waiting_on': wo}
 
     def get_triage_facts(self, issuewrapper, meta):
+        iw = issuewrapper
         tfacts = {
-            'maintainer_triaged': False
+            'maintainer_triaged': iw.issue.user.login in self.ansible_core_team,
```

Comment:
I don't think `iw.issue.user.login` exists. You probably want `iw.submitter` instead?

Reply:
Dunno, I've followed doc:
* https://pygithub.readthedocs.io/en/latest/github_objects/Issue.html (`iw.issue`)
* https://pygithub.readthedocs.io/en/latest/github_objects/Issue.html#github.Issue.Issue.user (`iw.issue.user`) -> https://pygithub.readthedocs.io/en/latest/github_objects/NamedUser.html
* https://pygithub.readthedocs.io/en/latest/github_objects/NamedUser.html#github.NamedUser.NamedUser.login (`iw.issue.user.login`)

And there's nothing called `submitter` in that doc.

Reply:
Oh, I see. Yes, `iw.submitter` from https://github.com/ansible/ansibullbot/blob/master/ansibullbot/wrappers/defaultwrapper.py#L853 should work.